### PR TITLE
LPS-179860 Remove feature flag

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

https://issues.liferay.com/browse/LPS-179860

Remove feature flag for user agent matching in redirections.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
@Hanlf Can you review the https://github.com/liferay-lima/liferay-portal/pull/4350/commits/8e7afcd299df008e85f1ea985911486cf13468f1 commit? I've updated the Poshi tests.

File: modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/util/PatternUtil.java
Comment (marcogalluzzi) [inline modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/util/PatternUtil.java:44]: There's no need for declaring variables `limit` and `userAgent`, they can be inlined right?

File: modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java
Comment (marcogalluzzi) [inline modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java:1]: Can we remove the `_props` attribute from this class?

**Review by marcogalluzzi — CHANGES_REQUESTED** [review]

**Comment by adolfopa** [comment]
@marcogalluzzi I applied your suggestions and squashed all changes.

**Review by marcogalluzzi — APPROVED** [review]

**Comment by marcogalluzzi** [comment]
Ooops I should not forward it. We still need a review from @Hanlf .

**Comment by marcogalluzzi** [comment]
ci:stop

**Comment by marcogalluzzi** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:test:help

**Comment by WYZ-Issac** [comment]
ci:test:redirect

**Comment by Hanlf** [comment]
Sorry for the late reply, the Poshi updates LGTM, waiting for Issac's final review

**Review by Hanlf — APPROVED** [review]

**Comment by WYZ-Issac** [comment]
ci:test:redirect

**Comment by WYZ-Issac** [comment]
Hi @adolfopa, after checking the test result, I found there are some backend tests that failed the first time in your PR. Could you have a look at these errors?
[RedirectPatternConfigurationModelListenerTest](https://testray.liferay.com/zh/home/-/testray/case_results/2326679348)
[StagingDataPortletPreferencesTest](https://testray.liferay.com/zh/home/-/testray/case_results/2326679191)
[PortalLogAssertorTest-modules-integration-mysql57-jdk8/3/23](https://testray.liferay.com/zh/home/-/testray/case_results/2326679107)
[empty-osgi-core-dir-mysql57-jdk8/0/0](https://testray.liferay.com/zh/home/-/testray/case_results/2326676823)

**Comment by adolfopa** [comment]
Fixed failing test

**Comment by adolfopa** [comment]
ci:forward

**Comment by Hanlf** [comment]
ci:forward

**Pull request closed without merging**
